src/test/scala/strawman/collection/test/TraverseTest.scala

```diff
@@ -69,9 +69,8 @@ class TraverseTest {
 
   @Test
   def eitherSequenceTest: Unit = {
-    val xs3 = mutable.ListBuffer(Right("foo"), Left(0), Right("bar"))
+    val xs3: mutable.ListBuffer[Either[Int, String]] = mutable.ListBuffer(Right("foo"), Left(0), Right("bar"))
```

Comment:
@ichoran That's not how implicit search works. We infer a type for certain type variables and then stick to it. If we selectively widen on failure we'd get another dimension of search complexity.

@smarter Yes, changing Either would work here, but I fear the problem is more general.